pom.xml

```diff
@@ -1417,6 +1439,7 @@ under the License.</licenseText>
       </activation>
       <properties>
         <spotless.action>apply</spotless.action>
+        <rewrite.action>run</rewrite.action>
```

Comment:
```suggestion
```
Yes we agree. The changes made should be checked in the dryRun goal and then see if they match the desired outcome.

With this its just warning but not changing. Agree whitespaces fixes are something else than actual code.